newIDE/app/src/InstancesEditor/InstancesRotator.js

```diff
@@ -1,46 +1,118 @@
 // @flow
+import Rectangle from '../Utils/Rectangle';
+
 export default class InstancesRotator {
-  instanceAngles: { [number]: number };
-  totalDeltaX: number;
-  totalDeltaY: number;
+  _instanceMeasurer: any;
+  _instanceAngles: { [number]: number } = {};
+  _instancePositions: { [number]: { x: number, y: number } } = {};
+  _instanceAABBs: { [number]: Rectangle } = {};
+  totalDeltaX: number = 0;
+  totalDeltaY: number = 0;
+  _anchorIsUpToDate: boolean = false;
```

Comment:
Could you explain what this anchor is doing exactly?

Reply:
It's the fixed point, here it's always the selection center but it could be made dragable one day.

Reply:
Thanks for the renaming!